docs/_docs/upgrading/0-to-2.md

```diff
@@ -1,140 +0,0 @@
----
-layout: docs
-title: Upgrading from 0.x to 2.x
-permalink: /docs/upgrading/0-to-2/
----
-
-Upgrading from an older version of Jekyll? A few things have changed in 1.0
```

Comment:
Let's leave this document – it's still useful. Believe it or not, there are plenty of folks using Jekyll 0.12.